src/plugins/intel_gpu/tests/test_cases/fully_connected_gpu_test.cpp

```diff
@@ -1006,7 +1006,9 @@ struct fully_connected_random_test : ::testing::TestWithParam<fully_connected_te
         auto bias = net.add_data<BiasT, 2>("bias", format::bfyx, std::move(bias_data));
         auto fc = net.add_fully_connected<OutputT>("fc_prim", input, weights, bias, ov::intel_gpu::ImplementationDesc{ output_format, kernel });
 
-        net.run(ExecutionConfig(ov::intel_gpu::optimize_data(true)), is_caching_test);
+        ExecutionConfig config = get_test_default_config(eng);
+        config.set_property(ov::intel_gpu::optimize_data(true));
```

Comment:
what about expanding `get_test_default_config` to receive additional vector of properties? If so, we don't need to separate like this. Instead, we can call like below:
`        net.run(get_test_default_config(eng, ov::intel_gpu::optimize_data(true)), is_caching_test);`

Reply:
Discussion : remove this line.
No reason to support and test in-order queue-type if oneDNN supports out-of-order.
it better remove in_order queue setting now. If thereis any special case for in-order queue test, then it should be handled in each unit-test when in-order queue is removed.